docs/Documentation/Applications/vasp.md

```diff
@@ -0,0 +1,537 @@
+VASP computes an approximate solution to the many-body Schrödinger equation, either within density functional theory (DFT), solving the Kohn-Sham equations, or within the Hartree-Fock (HF) approximation, solving the Roothaan equations. Hybrid functionals that mix the Hartree-Fock approach with density functional theory are implemented as well. Furthermore, Green's functions methods (GW quasiparticles, and ACFDT-RPA) and many-body perturbation theory (2nd-order Møller-Plesset) are available in VASP.
+
+In VASP, central quantities, like the one-electron orbitals, the electronic charge density, and the local potential are expressed in plane wave basis sets. The interactions between the electrons and ions are described using norm-conserving or ultrasoft pseudopotentials, or the projector-augmented-wave method.
+
+To determine the electronic ground state, VASP makes use of efficient iterative matrix diagonalization techniques, like the residual minimization method with direct inversion of the iterative subspace (RMM-DIIS) or blocked Davidson algorithms. These are coupled to highly efficient Broyden and Pulay density mixing schemes to speed up the self-consistency cycle.
+
+For further details, documentation, forums, and FAQs, see the [VASP website](https://www.vasp.at/).
+
+## Accessing VASP on NREL's HPC Clusters
+
+The VASP license requires users to be a member of a "workgroup" defined by the University of Vienna or Materials Design. If you are receiving "Permission denied" errors when trying to use VASP, you must be made part of the "vasp" Linux group first. To join, please contact [HPC Help](mailto:hpc-help@nrel.gov) with the following information:
+
+- Your name
+- The workgroup PI
+- Whether you are licensed through Vienna (academic) or Materials Design, Inc. (commercial)
+- If licensed through Vienna:
+    - The e-mail address under which you are registered with Vienna as a workgroup member (this may not be the e-mail address you used to get an HPC account)
+    - Your VASP license ID
+- If licensed through Materials Design:
+    - Proof of current licensed status
+
+Once status can be confirmed, we can provide access to our VASP builds. 
+
+## Supported Versions
+
+NREL offers support for VASP 5 and VASP 6 on CPUs as well as GPU builds on certain systems. (See table below for details).
+
+- For CPUs, VASP can be built with either Intel compilers/Intel MPI or GNU compilers/Open MPI. In general the Intel MPI builds exhibit fastest performance and are recommended over the Open MPI builds.
+
+- For GPUs, VASP can be built with Cuda or with OpenACC. The OpenACC GPU-port of VASP was released with VASP 6.2.0, and the Cuda GPU-port of VASP was dropped in VASP 6.3.0. The OpenACC build shows significant performance improvement compared to the Cuda build, but is more susceptible to running out of memory. 
+
+|                    |     Eagle     |     Swift     |   Vermilion   |
```

Comment:
Do you know yet which versions will be available on Kestrel?

Reply:
No not yet, we will need to revise when the builds are ready on Kestrel and add some more sample run scripts